metrics-core/src/main/java/com/codahale/metrics/SimpleSettableGauge.java

```diff
@@ -0,0 +1,44 @@
+package com.codahale.metrics;
+
+/**
+ * <p>
+ * Similar to {@link Gauge}, but metric value is updated via calling {@link #setValue(T)} instead.
+ * Thread-safety is ensured via storing metric value as a volatile member variable.
+ * </p>
+ */
+public class SimpleSettableGauge<T> implements SettableGauge<T> {
+    /**
+     * Current value.
+     * Volatile so that assignment is thread-safe.
```

Comment:
You don't need to explain why volatile is there (people should know it). Probably the whole field comment is unnecessary.